main/contracts/erc20.ts

```diff
@@ -65,6 +65,7 @@ export default class Erc20Contract {
         symbol: calls[2]
       }
     } catch (e) {
+      console.log(`getTokenData error: ${e}`)
```

Comment:
console log

Reply:
I'll put this in the main logs because I think it's useful to log token data fetch errors